src/App.js

```diff
@@ -3,10 +3,11 @@ import "./scss/App.scss";
 import Display from "./Display";
 
 function App() {
+    const windowHeight = window.screen.height;
```

Comment:
Maybe you want some destructuring?
`const { screen: { height: windowHeight } } = window;`